code/__HELPERS/_auxtools_api.dm

```diff
@@ -2,8 +2,12 @@
 #define AUXTOOLS_PARTIAL_INIT 1
 
 GLOBAL_LIST_EMPTY(auxtools_initialized)
+GLOBAL_PROTECT(auxtools_initialized)
 
 #define AUXTOOLS_CHECK(LIB)\
+	if (!CONFIG_GET(flag/auxtools_enabled)) {\
+		CRASH("Auxtools is not enabled in config!");\
```

Comment:
This is really annoying, throws an error any time I try and debug anything.
Was that intended? why is this an error?